### ISSUE TITLE
Zend Engine API warning when using Valet+

### ISSUE THREAD

**Issue by jaap** [weprovide/valet-plus#227]
When firing up valet I get the warning;

```
Xdebug requires Zend Engine API version 320170718.
The Zend Engine API version 320160303 which is installed, is outdated.
```

Running `valet xdebug on` tells me

```
Xdebug not found, installing...
```

Which makes me think my installation has got some legacy because of my former installation.

At this moment I'm not running into any issues, it just feels a bit dodgy. If anyone knows how to clean this up, help would be much appreciated.

**Comment by samgranger** [comment]
Which PHP & Valet+ version are you running?

**Comment by jaap** [comment]
Fixed! Php 7.1 was still loading `/usr/local/etc/php7.1/ext-xdebug.ini` which was probably a left-over of my previous journey...

Removing the configuration finnaly fixed everything, with Xdebug running smooth as butter with VS Code. Thanks anyway @samgranger .
